WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -84,6 +77,24 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
     return [super hitTest:point withEvent:event];
 }
 
+- (void)didMoveToSuperview
+{
+    [super didMoveToSuperview];
+    if (self.didPreserveStartingConstraintConstants) {
```

Comment:
`didMoveToSuperview` gets called when removing a view from its superview too :) 
It should be fine since the constants are updated when added and we've done the necessary work before any subsequent removal.